das/daser.go

```diff
@@ -41,7 +41,7 @@ type DASer struct {
 
 	cancel         context.CancelFunc
 	subscriberDone chan struct{}
-	running        int32
```

Comment:
Do you mind to change this one to use atomic.Bool? It suites the usecase better here

Reply:
Pretty valid suggestion, fixed & thanks!